src/logion/model/locrequest.model.ts

```diff
@@ -289,6 +296,7 @@ export class LocRequestAggregateRoot {
 }
 
 @Entity("loc_request_file")
+@Unique("UQ_request_id_hash", ["requestId", "hash"])
```

Comment:
Do not specify a name for the constraint,  it will then delegate name generation to `LogionNamingStrategy`.